bot/cogs/error_handler.py

```diff
@@ -87,9 +88,29 @@ def __init__(self, bot: Bot):
                     return
 
                 # Return to not raise the exception
-                with contextlib.suppress(ResponseCodeError):
-                    await ctx.invoke(tags_get_command, tag_name=ctx.invoked_with)
+                tags_cog = self.bot.get_cog("Tags")
+                sent = await tags_cog._get_command(ctx, command_name)
+                # sent = await tags_get_command.callback(tags_get_command.cog, ctx, ctx.invoked_with)
+                if sent:
                     return
+                # No similar tag found, or tag on cooldown -
+                # searching for a similar command
+                raw_commands = [
+                    (cmd.name, *cmd.aliases)
+                    for cmd in self.bot.walk_commands()
+                    if not cmd.hidden
+                ]
+                raw_commands = [c for data in raw_commands for c in data]
```

Comment:
Perhaps this could be cached but it seems a bit complicated to keep it up to date considering commands can be added/removed during runtime (e.g. via unloading extensions). Not sure if worth looking into.

Reply:
I will note it :ok_hand: